cookbooks/bcpc-hadoop/recipes/hive_table_stat.rb

```diff
@@ -30,7 +32,7 @@
     if not system " #{cmd} 'SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = #{node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db"]}' | grep -q #{node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db"]}" then
       code = <<-EOF
         CREATE DATABASE #{node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db"]};
-        GRANT #{privs} ON #{node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db"]}.* TO '#{node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db_user"]}'@'%' IDENTIFIED BY '#{get_config('mysql-hive-table-stats-password')}';
+        GRANT #{privs} ON #{node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db"]}.* TO '#{node["bcpc"]["hadoop"]["hive"]["hive_table_stats_db_user"]}'@'%' IDENTIFIED BY '#{get_config('password', 'mysql-hive-table-stats', 'os')}';
         EOF
       IO.popen("mysql -uroot -p#{get_config('mysql-root-password')}", "r+") do |db|
```

Comment:
This should coordinate with the Chef-BACH #260 MySQL password vaulting; either Chef-BACH #260 should update this code or this code should use its password convention.